src/base-apis.js

```diff
@@ -412,6 +412,30 @@ MatrixBaseApis.prototype.getGroupSummary = function(groupId) {
 };
 
 /**
+ * @param {string} groupId
+ * @param {Object} profile The group profile object, with name, avatar_url,
```

Comment:
Perhaps specify the individual options in the same style as https://github.com/matrix-org/matrix-js-sdk/pull/512/files#diff-7ecbd1742c9fa46478efe534d035eb28R374